inc/class-taxonomies-table.php

```diff
@@ -439,5 +440,23 @@ protected function column_count($item)
         return $title;
     }
 
+    /**
+     * The edited_with_taxopress column
+     *
+     * @param object $item
+     * @return string
+     */
+    protected function column_edited_with_taxopress($item)
+    {
+        $external_taxonomies = get_option('taxopress_external_taxonomies', array());
+        $taxopress_taxonomies = taxopress_get_taxonomy_data();
+        
+        if (array_key_exists($item->name, $taxopress_taxonomies) || array_key_exists($item->name, $external_taxonomies)) {
+            return '<span class="dashicons dashicons-yes-alt taxopress-edited-indicator taxopress-edited-yes" title="' . esc_attr__('Edited with TaxoPress', 'simple-tags') . '"></span>';
```

Comment:
@Deji98 I thought we agreed on adding tooltip to the icon with message on this taxonomy has been edited with taxopress for edited taxonomy and not edited with taxopress for those that haven't been edited?

Reply:
or are you referring to the title attribute you added to the icon?